cms/djangoapps/contentstore/config/waffle_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Util methods for Waffle checks"""
+
+from cms.djangoapps.contentstore.config.waffle import waffle, ENABLE_CHECKLISTS_PAGE, ENABLE_CHECKLISTS_QUALITY
+# from courseware.access import has_access
+from student.roles import GlobalStaff
+
+
+def should_show_checklists_page(requesting_user):
+    """
+        Determine if the COURSE_HEALTH_PAGE waffle is set
```

Comment:
Nit: `COURSE_HEALTH_PAGE` waffle flag was renamed to `ENABLE_CHECKLISTS_PAGE`, and maybe specify waffle flag instead of just waffle.